NEWS.md

```diff
@@ -1,3 +1,9 @@
+#moderndive 0.5.3.9002
```

Comment:
Version number is inconsistent with the one in the DESCRIPTION file. 9002 vs 9003.